engine/crates/integration-tests/tests/graphql_connector/errors.rs

```diff
@@ -334,3 +334,84 @@ fn graphql_connector_error_propagation_unnamespaced_no_grouping() {
         );
     });
 }
+
+#[test]
+fn aliases_on_unnamespaced_queries() {
```

Comment:
If this is testing error handling it seems like a bad test name

Reply:
oups, was copy-pasted 😅